### PR TITLE
Add Initial tables

### PR DESCRIPTION
**Pull request by bigdatasourav**

### CODE REVIEW

**Review by madhushreeray30 — CHANGES_REQUESTED** [review]
Please have a look at the comments. Thanks!

File: azuredevops/table_azuredevops_user.go [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "origin_id",
+				Description: "The unique identifier from the system of origin. Typically a sid, object id or Guid. Linking and unlinking operations can cause this value to change for a user because the user is not backed by a different provider and has a different unique id in the new provider.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "subject_kind",
+				Description: "This field identifies the type of the graph subject (ex: User, Scope, User).",
```
[the remark this answers is not in the document]
Comment (bigdatasourav) [reply azuredevops/table_azuredevops_user.go:89]: Updated.

File: docs/tables/azuredevops_user.md
[the remark this answers is not in the document]
Comment (bigdatasourav) [reply docs/tables/azuredevops_user.md:1]: Added the above one.

File: docs/tables/azuredevops_team_member.md
[the remark this answers is not in the document]
Comment (bigdatasourav) [reply docs/tables/azuredevops_team_member.md:1]: Added the above.

**Comment by misraved** [comment]
@bigdatasourav did we consider other authentication methods like SSH per https://learn.microsoft.com/en-us/azure/devops/repos/git/auth-overview?view=azure-devops#authentication-comparison

**Comment by bigdatasourav** [comment]
@misraved As of now the [SDK](https://github.com/microsoft/azure-devops-go-api/) supports personal access token. We can always add more authentication modes once they are available in SDK.

**Pull request merged**